.github/workflows/test.yaml

```diff
@@ -47,7 +49,7 @@ jobs:
           python-version: "${{ matrix.python-version }}"
       - uses: actions/setup-node@v4
         with:
-          node-version: "18"
+          node-version: "lts/*"
```

Comment:
In order to reduce maintenance, where we expect things to keep working when new lts versions come anyhow.